src/main/java/org/jenkinsci/test/acceptance/plugins/ldap/SearchForGroupsLdapGroupMembershipStrategy.java

```diff
@@ -5,7 +5,7 @@
 /**
  * @author Michael Prankl
  */
-@Describable("Search for groups containing user")
+@Describable("Search for LDAP groups containing user")
```

Comment:
Please preserve both of the captions not to break the ATH for older versions - the annotation accepts list of values.